openapi/components/schemas/webhook-dispatches/WebhookDispatch.yaml

```diff
@@ -28,12 +28,10 @@ properties:
     type: string
     example: ACTOR.RUN.SUCCEEDED
   eventData:
-    allOf:
-      - $ref: ./eventData.yaml
-      - {}
+    $ref: ../eventData.yaml
```

Comment:
How about renaming this as well?

Reply:
Inlined these two, they were not used elsewhere